src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, isMarried, sex, partnerName } = person;
+  let partnerStatus = 'I am not married';
+
+  if (isMarried) {
+    if (sex === 'f') {
+      partnerStatus = `${partnerName} is my husband`;
+    } else {
+      partnerStatus = `${partnerName} is my wife`;
+    }
```

Comment:
It's beautiful "if / else" logic, but in my opinion, there would be a better version of the code, shorter, like this:

```suggestion
    partnerStatus = `${name} is my ${sex === 'f' ? 'husband' : 'wife'}`;
```